core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -965,7 +1065,7 @@ private void processWorkerMetrics(String hostname, List<Metric> metrics) {
    * @param workerInfo The worker metadata object
    * @param removedBlockIds A list of block ids removed from the worker
    */
-  @GuardedBy("workerInfo")
+  @GuardedBy("workerInfo.mBlockListLock")
```

Comment:
Do I refer to the field or getter method `workerInfo.getBlockListLock()`?

Reply:
I didn't know you could attach this to a method. 
Honestly, it doesn't make sense to me.

Reply:
Cool. Removed from the `DefaultBlockMaster` methods